### PR TITLE
Alek

### PR DESCRIPTION
**Pull request by adschaf**
The tabs we created for index.js still will not find the pages. I confused myself and have no clue what to do from here. I have made sure what needs to be added and what does not but I cannot figure out the pages needed to do this. @markvoortman 
I have also check each and every page for what needs to be found but we have so many random pages that we do not use and it's making me confused.

### CODE REVIEW

**Comment by othot-mvoortman** [comment]
I think you did it correctly. Did you restart node?

File: index.js [hunk 1]
```diff
@@ -18,70 +18,68 @@ app.engine("handlebars",handlebars.engine);
...
 // secret:credentials.cookieSecret
 //}));
+
 //app.get("/", function(req,res){
 //    res.render("home");
 //});
+
+function getMenu(req) {
+  var menu = [];
+  var isAdmin = true;
```
Comment (othot-mvoortman) [inline index.js]: The `isAdmin` information should come from the session. Why was this not modified yet? Also, there should be a difference between being logged in and logged out and that is also not implemented yet.

Comment (adschaf) [reply index.js]: I have got the sign in to work but I could not get the log in to work. I thought we created the isAdmin right there or do I need to create it somewhere else?

**Comment by bigmatal87** [comment]
To all.
    I'm not going to be able to make either class today. I have to go deal with a fighting issue at my kids school.


Matt
________________________________

**Pull request merged**